test/python/evaluation/testing_config.py

```diff
@@ -0,0 +1,12 @@
+import argparse
+from typing import NoReturn
+
+from src.python.review.reviewers.perform_review import OutputFormat
+
+
+def get_parser() -> NoReturn:
```

Comment:
Why does this function have the return type `NoReturn` if the function returns `ArgumentParser`?

Reply:
Indeed, resolved. Thank you!